release notes/v0.40.0.md

```diff
@@ -0,0 +1,280 @@
+- [#2596](https://github.com/grafana/k6/pull/2596) The `tainted` property of the Metric type is no longer outputted by the JSON output. That property was likely always going to have a `false` value as it was outputted at the beginning of the test.
+
+## Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)
+
+During the ESM changes, we found that anything defined in the main module scope was also accessible globally. This was because it was directly evaluated in the global scope.
+This has now been remedied and is no longer the case. This is a *breaking* change, but given that the whole point of modules (CommonJS or ESM) is to separate them, this is obviously rather a bug than a feature.
+
+On that note, we've seen [reports](https://github.com/grafana/k6/issues/2623) by people who have this global accessibility of the main module (intentionally or not). Still, it seems relatively rare, with only a few usages in a script. So if you need to access something globally, our suggested workaround is to set it explicitly on the global object `globalThis`.
+
+## `k6/ws` now considers the `throw` option [#2247](https://github.com/grafana/k6/issues/2247)
+
+`k6/http` has used the [`throw` option](https://k6.io/docs/using-k6/k6-options/reference/#throw) to figure out whether it should throw an exception on errors or return a response object with an error set on it (and log it).
+
+This functionality is finally also available for `k6/ws`, which previously would've *always* thrown an exception and thus involved more scripting to handle it.  [#2616](https://github.com/grafana/k6/pull/2616)
+
+This is a minor breaking change. By default, `throw` is `false`, so it now no longer throws an exception but instead returns a Response with `error` property.
+
+Thank you, @fatelei, for making this change !
+
+# New Features
+
+## Experimental modules [#2630](https://github.com/grafana/k6/pull/2630) and [#2656](https://github.com/grafana/k6/pull/2656)
+
+As mentioned in the v0.39.0 release notes, we're happy to announce that this release brings experimental modules. The main idea behind this initiative is to get community feedback earlier, which will help us improve them faster. We encourage you to try experimental modules out and provide feedback through the community forums or GitHub issues. 
+
+This release contains three experimental modules:
+- Support for interaction with [Redis](https://redis.io/)
+- A new Websockets API that copies the "web" [WebSocket API](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket)
+- [setTimeout](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout) and friends.
```

Comment:
Should we maybe avoid "friends" since it is the too low-context term :thinking: ? 

```suggestion
- [`setTimeout`](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout)/[`clearTimeout`](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout) and [`setInterval`](https://developer.mozilla.org/en-US/docs/Web/API/setInterval)/[`clearInterval`](https://developer.mozilla.org/en-US/docs/Web/API/clearInterval) implementations.
```

Reply:
I agree. It also sounds very anthropomorphic and informal, like it's a cartoon:

![image](https://user-images.githubusercontent.com/47385188/188437488-9e6e1e0c-bcea-47d0-a8cb-3f48b52b2835.png)